api/coreservice.go

```diff
@@ -426,7 +426,9 @@ func (core *coreService) SendAction(ctx context.Context, in *iotextypes.Action)
 	}
 	// If there is no error putting into local actpool,
 	// Broadcast it to the network
-	if err = core.broadcastHandler(ctx, core.bc.ChainID(), in); err != nil {
+
+	msg := &iotextypes.Actions{Actions: []*iotextypes.Action{in}}
```

Comment:
This commit is also cherry-picked by test image. The switch is turned on temporally. The change will be reverted before merged